orthoinference/src/main/java/org/reactome/orthoinference/GenerateInstance.java

```diff
@@ -0,0 +1,200 @@
+package org.reactome.orthoinference;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+import org.gk.schema.GKSchemaAttribute;
+import org.gk.schema.GKSchemaClass;
+import org.gk.schema.InvalidAttributeException;
+import org.gk.schema.InvalidAttributeValueException;
+import org.gk.schema.SchemaClass;
+
+
+// GenerateInstance is meant to act as a catch-all for functions that are instance-oriented, such as creating, mocking, or identical-checking.
+public class GenerateInstance {
```

Comment:
Maybe rename this to "InstanceUtilities" - many methods in here don't seem to have anything to do with "generating" anything.

Reply:
Yea thats what I was thinking actually, it started out as an 'instance-specific' class, but gradually became more of a utility class